sdk/src/test/java/io/opentdf/platform/sdk/ZipReaderTest.java

```diff
@@ -0,0 +1,63 @@
+package io.opentdf.platform.sdk;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.RandomAccessFile;
+import java.nio.ByteBuffer;
+import java.nio.channels.FileChannel;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+public class ZipReaderTest {
+    private ZipReader zipReader;
+    private ByteBuffer buffer;
+    private RandomAccessFile raf;
+    private FileChannel fileChannel;
+
+
+    @Test
+    public void testZipReader() throws Exception {
+        RandomAccessFile raf = new RandomAccessFile("src/test/resources/sample.txt.tdf", "r");
```

Comment:
maybe read this as a resource so that it's insensitive to the working directory?